nilearn/input_data/nifti_masker.py

```diff
@@ -199,8 +199,8 @@ def fit(self, imgs=None, y=None):
             self.affine_ = self.mask_img_.get_affine()
         # Load data in memory
         self.mask_img_.get_data()
-        if self.verbose > 10:
-            print "[%s.fit] Finished fit" % self.__class__.__name__
+        if self.verbose > 0:
```

Comment:
I am guessing this change (verbose > 10 rather than verbose > 0) wasn't meant to be there